src/librawspeed/metadata/CameraMetaData.cpp

```diff
@@ -67,10 +67,6 @@ CameraMetaData::~CameraMetaData() {
 
 static inline string getId(string make, string model, string mode)
 {
-  TrimSpaces(make);
```

Comment:
Now this is just wrong.
You do see that it is called from `addCamera()`, right?